specs-go/config.go

```diff
@@ -40,13 +40,12 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
-	// Capabilities are Linux capabilities that are kept for the container.
+	// Capabilities are Linux capabilities that are kept for the container. (this field is platform dependent)
 	Capabilities []string `json:"capabilities,omitempty" platform:"linux"`
-	// Rlimits specifies rlimit options to apply to the process.
-	Rlimits []Rlimit `json:"rlimits,omitempty"`
-	// NoNewPrivileges controls whether additional privileges could be gained by processes in the container.
-	NoNewPrivileges bool `json:"noNewPrivileges,omitempty"`
-
+	// Rlimits specifies rlimit options to apply to the process. (this field is platform dependent)
+	Rlimits []Rlimit `json:"rlimits,omitempty" platform:"linux,solaris"`
+	// NoNewPrivileges controls whether additional privileges could be gained by processes in the container. (this field is platform dependent)
+	NoNewPrivileges bool `json:"noNewPrivileges,omitempty" platform:"linux,solaris"`
```

Comment:
I don't think that these `Rlimits` or `NoNewPrivileges` are on Solaris.  And we can probably wait for Solaris folks to pipe up if they are, instead of adding them here and waiting for Solaris folks to pipe up to re-remove them ;).